build.sbt

```diff
@@ -33,8 +33,7 @@ lazy val compilerFlags = Seq(
     "-deprecation",
     "-encoding", "utf-8",
     "-feature",
-    "-language:higherKinds",
-    "-Ypartial-unification"
+    "-language:higherKinds"
```

Comment:
Was this on purpose?

Reply:
Yeah, 2.13 has partial unification on by default.

Reply:
Yup.